shared/google-maps/index.js

```diff
@@ -5,6 +5,10 @@ const prefix = 'http://www.google.com/maps/place/';
 const getCoordinates = pathOr({}, ['coordinates']);
 
 export function googleMapsUrl(location) {
+  // We don't want a maps url for a remote event
+  const address = pathOr('', ['address'], location);
+  if (address.toLowerCase() === 'remote') return null;
```

Comment:
Might want to extract these lines to a util as they're used in two different files.